aws/data_source_aws_efs_file_system.go

```diff
@@ -128,6 +132,7 @@ func dataSourceAwsEfsFileSystemRead(d *schema.ResourceData, meta interface{}) er
 	d.Set("kms_key_id", fs.KmsKeyId)
 	d.Set("provisioned_throughput_in_mibps", fs.ProvisionedThroughputInMibps)
 	d.Set("throughput_mode", fs.ThroughputMode)
+	d.Set("size_in_bytes", fs.SizeInBytes.Value)
```

Comment:
Nit: To prevent potential panics, this may want a nil check:

```suggestion
	if fs.SizeInBytes != nil {
		d.Set("size_in_bytes", fs.SizeInBytes.Value)
	}
```

Reply:
On it.